src/module_2/module_2_EDA.md

```diff
@@ -0,0 +1,2197 @@
+     3   first_ordered_at       4983 non-null   object 
+     4   customer_cohort_month  4983 non-null   object 
+     5   count_people           325 non-null    float64
+     6   count_adults           325 non-null    float64
+     7   count_children         325 non-null    float64
+     8   count_babies           325 non-null    float64
+     9   count_pets             325 non-null    float64
+    dtypes: float64(5), object(5)
+    memory usage: 428.2+ KB
+    
+    
+    Número de valores null:
+    user_id                     0
+    user_segment                0
+    user_nuts1                 51
+    first_ordered_at            0
+    customer_cohort_month       0
+    count_people             4658
+    count_adults             4658
+    count_children           4658
+    count_babies             4658
+    count_pets               4658
+    dtype: int64
+    
+    
+    Porcentaje de valores null:
+    user_id                   0.000000
+    user_segment              0.000000
+    user_nuts1                1.023480
+    first_ordered_at          0.000000
+    customer_cohort_month     0.000000
+    count_people             93.477825
+    count_adults             93.477825
+    count_children           93.477825
+    count_babies             93.477825
+    count_pets               93.477825
+    dtype: float64
+    
+    
+
+
+
+```python
+df_users.groupby('user_segment').count()
+```
+
+
+
+
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>user_id</th>
+      <th>user_nuts1</th>
+      <th>first_ordered_at</th>
+      <th>customer_cohort_month</th>
+      <th>count_people</th>
+      <th>count_adults</th>
+      <th>count_children</th>
+      <th>count_babies</th>
+      <th>count_pets</th>
+    </tr>
+    <tr>
+      <th>user_segment</th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Proposition</th>
+      <td>2340</td>
+      <td>2315</td>
+      <td>2340</td>
+      <td>2340</td>
+      <td>193</td>
+      <td>193</td>
+      <td>193</td>
+      <td>193</td>
+      <td>193</td>
+    </tr>
+    <tr>
+      <th>Top Up</th>
+      <td>2643</td>
+      <td>2617</td>
+      <td>2643</td>
+      <td>2643</td>
+      <td>132</td>
+      <td>132</td>
+      <td>132</td>
+      <td>132</td>
+      <td>132</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+Más del 93% de los datos en 5 categorias son Null, parece que se dejaron de pedir por algun motivo (quizas temas de privacidad) o que se han pedido de manera opcional y pocos usuarios los proporcionan.
```

Comment:
indeed these is personal data about the users that they are free to fill or not, that's why most of them are missing